app/scripts/contentscript.js

```diff
@@ -83,6 +85,28 @@ function createCommitChart(){
     myChart.setOption(option)
 }
 
+/**
+ * Create check model for the report element.
+ */
+function createCheckModel(text, boolCheck){
+    var content = document.getElementsByClassName("new-discussion-timeline experiment-repo-nav")
+    var repoContent = document.getElementsByClassName("repository-content")
+    var node = document.createElement('div')
+    var title = document.createElement('H2')
+    title.style = ('padding-left: 400px')
```

Comment:
Why there is a `padding-left: 400px`?
The check model will be called in many places, not only on a 400px offset from left.

Please, center the title properly.

Reply:
I recommend the read: https://www.w3schools.com/css/css_align.asp